cluster-up/cluster/kind/common.sh

```diff
@@ -25,6 +25,7 @@ export KIND_MANIFESTS_DIR="${KUBEVIRTCI_PATH}/cluster/kind/manifests"
 export KIND_NODE_CLI="docker exec -it "
 export KUBEVIRTCI_PATH
 export KUBEVIRTCI_CONFIG_PATH
+KIND_DEFAULT_NETWORK="kind"
```

Comment:
Since rest of the PR has support for dynamic kind network,
worth please to move this var to set_kind_params, rename it and set it as export with default value,
so it will be easy to change it if needed

Reply:
Its not common to configure the network comparing to the kind binary version and the nodes image
In any case if we are going to do it I prefer that we do it on a follow up PR :slightly_smiling_face:

Reply:
It is possible and common and this is why the new kind supports custom network name in parameter
imho its just a few lines and suits here, as this PR added this parameter
i won't insist anyhow

Reply:
> It is possible and common

I have not seen this used much. We don't advertise it. And`KIND_EXPERIMENTAL_*` is not considered stable.

Not common on github at all: https://grep.app/search?q=KIND_EXPERIMENTAL_DOCKER_NETWORK